static/styles/subscriptions.css

```diff
@@ -141,22 +141,6 @@
     margin-right: 10px;
 }
 
-.preview-stream {
-    display: none;
-    float: right;
-    padding: 3px 10px;
-    border: 1px solid hsl(0, 0%, 80%);
-    margin: 9px 10px 0 0;
-    color: hsl(0, 0%, 20%);
-}
-
-.preview-stream:hover {
-    color: hsl(0, 0%, 20%);
-    text-decoration: none;
-    background-color: hsl(0, 0%, 92%);
-    border: 1px solid hsl(0, 0%, 68%);
-}
-
```

Comment:
Removed this `.preview-stream` as there do not exists any such classname/id in the code 

![Screenshot from 2022-02-27 23-08-09](https://user-images.githubusercontent.com/76561593/155896080-fe2f8fb2-5052-44f8-8f7c-244592042417.png)